src/ui-test/utils/testUtils.ts

```diff
@@ -14,20 +14,74 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import {
+	BottomBarPanel,
+	By,
+	ComboSetting,
 	ContentAssistItem,
 	EditorView,
 	InputBox,
+	Marketplace,
 	ModalDialog,
 	ProblemsView,
+	TerminalView,
 	TextEditor,
+	TextSetting,
+	WebDriver,
 	Workbench
 } from "vscode-uitests-tooling";
+import { storageFolder } from "../uitest_runner";
+import * as path from 'path';
+import * as fs from 'fs-extra';
+
+// Resources and file names inside.
+export const RESOURCES: string = path.resolve('src', 'ui-test', 'resources');
+export const CAMEL_CONTEXT_JAVA = 'camel-context.java';
+export const CAMEL_CONTEXT_XML = 'camel-context.xml';
+export const CAMEL_CONTEXT_YAML = 'camel-context.yaml';
+export const CAMEL_ROUTE_XML = 'camel-route.xml';
+
+export const XML_URI_LINE = 3; //camel-context.xml
+export const XML_URI_POSITION = 33; //camel-context.xml
+
+export const JAVA_URI_LINE = 9; //camel-context.java
+export const JAVA_URI_POSITION = 15; //camel-context.java
+
+export const YAML_URI_LINE = 7; //camel-context.yaml
+export const YAML_URI_POSITION = 14; //camel-context.yaml
+
+// Commands for creating Camel Routes.
+export const CREATE_COMMAND_XML = 'camel.jbang.routes.xml';
+export const CREATE_COMMAND_YAML = 'camel.jbang.routes.yaml';
+export const CREATE_COMMAND_JAVA = 'camel.jbang.routes.java';
+export const CREATE_COMMAND_QUARKUS = 'camel.jbang.project.quarkus.new';
+export const CREATE_COMMAND_SPRINGBOOT = 'camel.jbang.project.springboot.new';
+
+// Resources for testing.
+export const RESOURCES_COMMAND: string = path.resolve('src', 'ui-test', 'resources', 'camel_route_command');
+export const COMMAND_JAVA_FILE = 'Java.java';
+export const COMMAND_XML_FILE = 'XML.xml';
+export const COMMAND_YAML_FILE = 'YAML.yaml';
+
+// Identifiers of user preferences inside settings.json.
+export const JBANG_VERSION_ID = 'camel.languageSupport.JBangVersion';
+export const CATALOG_PROVIDER_ID = 'camel.Camel catalog runtime provider';
+export const CATALOG_VERSION_ID = 'camel.Camel catalog version';
+export const EXTRA_COMPONENTS_ID = 'camel.extra-components';
+
+export const JBANG_VERSION_UI = 'JBang Version';
+export const CATALOG_PROVIDER_UI = 'Camel catalog runtime provider';
+export const CATALOG_VERSION_UI = 'Camel catalog version';
+
+// Specific workspace for creating project with command.
+export const SPECIFIC_WORKSPACE: string = path.resolve('src', 'ui-test', 'resources', 'create-camel-project-workspace');
```

Comment:
maybe worthy to reuse already resolved path variable `RESOURCES` and just append by `path.join(RESOURCES, 'create-camel-project-workspace')` WDYT?